specification/components/schemas/Patient.yaml

```diff
@@ -77,4 +79,4 @@ properties:
         - $ref: extensions/DeathNotificationStatus.yaml
         - $ref: extensions/Communication.yaml
         - $ref: extensions/ContactPreference.yaml
-        - $ref: extensions/BirthPlace.yaml
\ No newline at end of file
+        - $ref: extensions/BirthPlace.yaml
```

Comment:
Why is BrithPlace.yaml removed and then re-added. could this be a mistake or a test??

Reply:
Think it's been pushed to a new line since ive added new lines